.changeset/eight-bees-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ponder/core": patch
+---
+
+Exported "alias" helper function.
```

Comment:
Exported Drizzle [`alias`](https://orm.drizzle.team/docs/joins#aliases--selfjoins)  function which can be used in API functions.